.kitchen.appveyor.yml

```diff
@@ -0,0 +1,23 @@
+---
+driver:
+  name: proxy
+  host: localhost
+  reset_command: "exit 0"
+  port: 5985
+  username: <%= ENV["machine_user"] %>
+  password: <%= ENV["machine_pass"] %>
+
+provisioner:
+  name: chef_zero
+
+platforms:
+  - name: windows-2012R2
+
+suites:
+- name: smoke_package_latest
+  run_list:
+  - recipe[jenkins_server_wrapper::default] 
```

Comment:
So right now this is going to fail because it includes java::default which assumes *nix. You can update that recipe to include the windows java recipe when on windows or java::default on any other platform. That might get this a bit closer to converging.